### PR TITLE
beta

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-11-05-apple-register-new-device-for-beta.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+image:
+image2:
+author: Valentino Urbano
+---
+
+I thought that registering a new device to use in internal betas would be breeze in 2019. It still isn't.
+
+## Easy Mode
+
+First there is the easy way where Xcode will do everything automatically. Unfortunately this method is only limited if all of the following points are satisfied:
```
Comment (MsSlacker) [inline _posts/2019-11-05-apple-register-new-device-for-beta.md]: limited - > available

File: _posts/2019-11-05-apple-register-new-device-for-beta.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+In this case is simply necessary to build and run the application and Xcode will automatically ask you if you want to add the device to the test devices if it is not already registered.
+
+Unfortunately when working on a client's project this is never the case. You would like to add the client's device so you can send betas without waiting for Beta App Review.
+
+## Manual Mode
+
+### Preparations For An Existing Account
+
+You might find yourself in front of the "reset your device list before adding any new devices" screen.
```
Comment (MsSlacker) [inline _posts/2019-11-05-apple-register-new-device-for-beta.md:31]: Missing paragraph (enter) ?

File: _posts/2019-11-05-apple-register-new-device-for-beta.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+Click reset, this will not delete any of your existing devices. In the next screen you will be able to select which one of your old devices you want to keep and which ones you want to delete.
+
+### Registering a new device
+
+Unfortunately iTunes is needed for this procedure.
+
+Connect the phone you want to add and open iTunes. Click on the device ID to show the UDID, right click to copy. Open the developer portal
```
Comment (MsSlacker) [inline _posts/2019-11-05-apple-register-new-device-for-beta.md]: Missing enter?

File: _posts/2019-11-05-apple-register-new-device-for-beta.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+[Look rest of process on slack to copy]
```
Comment (MsSlacker) [inline _posts/2019-11-05-apple-register-new-device-for-beta.md]: Look up? 

Maybe link?

File: _posts/2019-11-05-apple-register-new-device-for-beta.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+### Beta App Review
+
+If you just want to release a beta through Testflight you don't necessarely need to register a device. You can submit your app for beta app review and in that case, after approval, you will be able to add external accounts via email or even share a link to add testers without the need of adding devices through the portal at all. This requires a few more days waiting for App Review for the initial build, but on subsequent builds for the same release you do not need to go through beta app review again so it might be worth the few days wait in some cases.
```
Comment (MsSlacker) [inline _posts/2019-11-05-apple-register-new-device-for-beta.md]: necessarely necessarily

File: _posts/2019-11-05-apple-register-new-device-for-beta.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+## Conclusion
+
+It is usually best to just not add the device and wait for App Review if you're not so pressed for time. You could even upload a build some time before so that it is ready to go the first day you want to release for beta if you know the timetable and just release a minor update with the fixes and updates from then until now.
```
Comment (MsSlacker) [inline _posts/2019-11-05-apple-register-new-device-for-beta.md]: Run on!

"from then until now" sounds so bad, I don't even have an idea what it's supposed to mean.

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-11-05-apple-register-new-device-for-beta.md (hunk 4 above)
Comment (valeIT) [reply _posts/2019-11-05-apple-register-new-device-for-beta.md]: right...

File: _posts/2019-11-05-apple-register-new-device-for-beta.md (hunk 6 above)
Comment (valeIT) [reply _posts/2019-11-05-apple-register-new-device-for-beta.md]: From that moment onwards

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**